.github/workflows/docgen.yaml

```diff
@@ -1,30 +0,0 @@
-name: Generate documentation
-on: [push, pull_request]
-
-permissions:
-  contents: write
-
-jobs:
-  build:
-    name: Docgen
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        os:
-          - debian-bullseye
-
-    steps:
-      - uses: actions/checkout@v1
-      - name: Install deps
-        run: sudo apt-get -y --no-install-recommends install python3-markdown python3-blockdiag
-      - name: Run docgen
-        run: ./build_docs.py
-      - name: Upload to github pages
-        if: success()
-        uses: crazy-max/ghaction-github-pages@v1
-        with:
-          target_branch: gh-pages
-          build_dir: build_docs_output
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Does this mean that we're discontinuing the documentation at: https://api.ooni.io/?

If so, can we modify the content one last time to ensure there's a pointer to the new documentation?

Reply:
No, this was the autogenerated code that was published on some github pages link here: https://ooni.github.io/pipeline/README.html (it's referenced in the Readme, yet the link doesn't work and AFAIK has been broken for months and not updated for much more than that).

Reply:
✔️